node/runtime/pangolin/src/pallets/bridge_messages.rs

```diff
@@ -67,7 +63,7 @@ impl Config<WithPangolinParachainMessages> for Runtime {
 	type MaxUnrewardedRelayerEntriesAtInboundLane =
 		PangolinParachainMaxUnrewardedRelayerEntriesAtInboundLane;
 	type MessageDeliveryAndDispatchPayment =
-		FeeMarketPayment<Self, WithPangolinParachainFeeMarket, Ring, RootAccountForPayments>;
```

Comment:
Different versions of fee market?

Reply:
@AsceticBear

Reply:
Yes. In order to be consistence with the upstream repo, the newly fee market payment changed from [RootAccount](https://github.com/darwinia-network/darwinia-messages-substrate/blob/darwinia-v0.12.4/modules/fee-market/src/s2s/payment.rs#L56-L63)to  [submiter.linked_account](https://github.com/darwinia-network/darwinia-messages-substrate/blob/darwinia-v0.12.5/modules/fee-market/src/s2s/payment.rs#L60-L74) in this version.

Reply:
https://github.com/darwinia-network/darwinia-common/pull/1452